### ISSUE TITLE
Modals Malfunctioning (Only the Backdrop Appears)

### ISSUE THREAD

**Issue by DerekFoulk** [twbs/bootstrap#16109]
I just built bootstrap from the github repo today on LM17.1. The build was successful without any errors. So, I then copied the freshly built Bootstrap files from the '/dist' directory to my '/css' and '/js' directories respectively. The previous versions were successfully overwritten.

Now, the modals around my site are malfunctioning. The '.modal-backdrop.fade.in' div appears, but not the white box that houses the content.

I then reverted back to the Bootstrap files that I built a few days ago, and the modals work great again. I thought this to be odd, so I rebuilt Bootstrap and replaced the files with the newly built ones, and the modals are broken again... So, I believe this is a bug with Bootstrap that came to be in the last couple of days.

Please note: I have not touched the html during this debugging.

I have tested this on LM17.1 with Firefox (36.0.1), and Windows 7 with Firefox (35.0.1) & Chrome (41.0.2272.89 m). I also ensured that my cache was cleared after each test.

**Comment by DerekFoulk** [comment]
Oh, and I disabled my popup blocker (ABP) to make sure that wasn't causing any issues. To view the issue, see my site...
http://dev.nwca.derekfoulk.com/?page=product-listings
* Hover over a product and click the "QuickView" button.

**Comment by DerekFoulk** [comment]
I had to replace both the /css and /js directories to get the modals to function properly. If I just replaced the /js files, the modal content would appear, but it was rendered behind the tint.

**Comment by juthilo** [comment]
I don't see a problem with the modals behind any of your links using Chrome 41.0.2272.89. Did updating both dist directories solve your issue? Bootstrap's JS and CSS go hand in hand, so you'll always need update both.

Closing for now since I cannot reproduce.

**Comment by cvrebert** [comment]
This is due to you placing the modal within a `<div class="quick-view">`. The `display: none` style on your `.quick-view` class is making the modal invisible.
As stated in our docs, modals can't just be shoved into arbitrary places within your markup: http://getbootstrap.com/javascript/#modal-markup-placement

**Comment by DerekFoulk** [comment]
That makes sense. I have the '.quick-view' set to 'display:none;' until the parent element is hovered over. It definately returns to 'display:none;' on mouse-out. The only thing that is left unexplained is how replacing the bootstrap files was the only thing that I did, and they worked fine with the same html (thus the '.quick-view' was still set to 'display:none;'). I guess it doesn't matter. I will extract the modal markup and place it elsewhere. I did not see that notation in your documentation. I will definitely give it another read. Sorry for polluting your tracker lol. I honestly was convinced it was a legit bug (since swapping the files with older versions seemed to remedy the issue).

Thanks guys!
